10-inference.Rmd

```diff
@@ -313,7 +303,7 @@ when we estimated the population mean
 with only `r round(100*abs(estimates$sample_mean - population_parameters$pop_mean)/population_parameters$pop_mean, 1)`% error.
 Let's visualize the population
```

Comment:
why? this figure is just sort of put here without any explanation

i should say that i like the idea of including something like this, but it needs to be very carefully explained. probably an entire paragraph of text dedicated to it

Reply:
good point! I added a short explanation